_pages/robot/2016.html

```diff
@@ -7,16 +7,14 @@
 - medieval
 - fixedbackground
 description: Our robot and the challenge for the 2016 season
+year: 2016
```

Comment:
I don't love this

Reply:
I think it's helpful if I just add the banners through an includes tag. Then you only have to change the year on the homepage to change the banners you want to display for a specific year.

Reply:
Oh, absolutely, but I feel like there's maybe a better way to do this than adding a property to the page.